code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -295,6 +295,8 @@
 	cooldown_timer = 1 MINUTES
 	plasma_cost = 200
 	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_CURE
+	var/long_distance_heal = FALSE
```

Comment:
```suggestion
	var/ignore_distance_check = FALSE
```
would be a little clearer, and makes the checks later a little easier to understand.

Reply:
Done!